### PR TITLE
Feature/hint message color

### PR DESCRIPTION
**Pull request by LighthouseKeeperYN**
close: #1325

## Description
Add slot for messages in input-wrapper component.
Rework default message-list color from inline style to css variable

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-input/components/VaInputWrapper.vue [hunk 1]
```diff
@@ -80,6 +80,7 @@
     </div>
 
     <div class="va-input-wrapper__message-list-wrapper">
+      <slot name="messages" />
```
Comment (m0ksem) [inline packages/ui/src/components/va-input/components/VaInputWrapper.vue:83]: Let user completely redefine messages look. Please, check if VaMessageList is exposed in vuestic-ui.

```suggestion
      <slot name="messages" v-bind="{ messages: messagesComputed, errorLimit, color: messagesColor }">
            <va-message-list
                :color="messagesColor"
                :model-value="messagesComputed"
                :limit="errorLimit"
              />
      </slot>
```

**Review by m0ksem — CHANGES_REQUESTED** [review]
Thanks, please notice my suggestion.

**Review by m0ksem — APPROVED** [review]

**Review by RVitaly1978 — COMMENTED** [review]
Looks good.
Please, add `VaMessageList` to `src/vuestic-plugin/vuestic-components.ts`

File: packages/ui/src/components/va-input/_variables.scss [hunk 2]
```diff
@@ -7,6 +7,7 @@
   --va-input-border-radius: 4px;
   --va-input-min-height: 36px;
   --va-input-cursor: text;
+  --va-message-list-color: (--va-gray);
```
Comment (asvae) [inline packages/ui/src/components/va-input/_variables.scss:10]: var skipped :)
--va-message-list-color: var(--va-gray);

File: packages/ui/src/components/va-input/components/VaInputWrapper.vue [hunk 3]
```diff
@@ -205,6 +207,8 @@ export default defineComponent({
   }
 
   &-wrapper__message-list-wrapper {
+    vertical-align: middle;
```
Comment (asvae) [inline packages/ui/src/components/va-input/components/VaInputWrapper.vue:210]: Just curious, what's this for? :)

**Review by asvae — APPROVED** [review]
Thanks! Great work.
Feel free to merge once you're happy with state.

**Pull request merged**